classes/abstracts/ActionScheduler_TimezoneHelper.php

```diff
@@ -52,13 +54,13 @@ protected static function get_local_timezone_string( $reset = false ) {
 		}
 
 		// Get UTC offset, if it isn't set then return UTC.
-		$utc_offset = intval( get_option( 'gmt_offset', 0 ) );
-		if ( 0 === $utc_offset ) {
+		$utc_offset = get_option( 'gmt_offset', 0 );
+		if ( ! is_numeric( $utc_offset ) || 0 === $utc_offset ) {
```

Comment:
Settings the timezone to UTC will cause it to fail:

![Screenshot from 2020-06-03 11-14-37](https://user-images.githubusercontent.com/1264099/83647577-6f072d80-a58b-11ea-89d7-969e8437f161.png)

Seems like better to change to:

```php
    $utc_offset = floatval( get_option( 'gmt_offset', 0 ) );
    if ( ! is_numeric( $utc_offset ) || 0.0 === $utc_offset ) {
```